app/models/user.rb

```diff
@@ -5,9 +5,21 @@ class User < ApplicationRecord
          :recoverable, :rememberable, :validatable,
          :omniauthable, omniauth_providers: [:facebook]
         
-  has_attached_file :avatar, styles: { medium: '300x300>', thumb: '100x100>' }
+  has_one_attached :avatar 
```

Comment:
Remove this trailing space (the one at the end of the line), sometimes with just a single space like this Git will detect it as a change.